--- auto_link.hpp.orig	2018-01-09 18:53:33.539465800 +0100
+++ auto_link.hpp	2018-04-25 21:48:49.495300400 +0200
@@ -156,11 +156,21 @@
      // vc11:
 #    define BOOST_LIB_TOOLSET "vc110"
 
-#  elif defined(BOOST_MSVC)
+#  elif defined(BOOST_MSVC) && (BOOST_MSVC < 1900)
 
      // vc12:
 #    define BOOST_LIB_TOOLSET "vc120"
 
+#  elif defined(BOOST_MSVC) && (BOOST_MSVC < 1910)
+
+     // vc14:
+#    define BOOST_LIB_TOOLSET "vc140"
+
+#  elif defined(BOOST_MSVC)
+
+     // vc14.1:
+#    define BOOST_LIB_TOOLSET "vc141"
+
 #  elif defined(__BORLANDC__)
 
      // CBuilder 6:
