--- boost_1_55_0.orig/boost/archive/iterators/transform_width.hpp	2013-04-28 10:07:27.000000000 -0600
+++ boost_1_55_0/boost/archive/iterators/transform_width.hpp	2014-02-22 23:31:29.519297361 -0700
@@ -30,6 +30,8 @@
 #include <boost/iterator/iterator_adaptor.hpp>
 #include <boost/iterator/iterator_traits.hpp>
 
+#include <algorithm>
+
 namespace boost { 
 namespace archive {
 namespace iterators {
--- boost_1_55_0.orig/boost/config/compiler/visualc.hpp	2013-10-25 15:57:04.000000000 -0600
+++ boost_1_55_0/boost/config/compiler/visualc.hpp	2014-02-22 23:26:26.875102113 -0700
@@ -171,6 +171,7 @@
 // C++11 features supported by VC++ 12 (aka 2013).
 //
 #if _MSC_FULL_VER < 180020827
+#  define BOOST_NO_CXX11_DECLTYPE_N3276
 #  define BOOST_NO_CXX11_DEFAULTED_FUNCTIONS
 #  define BOOST_NO_CXX11_DELETED_FUNCTIONS
 #  define BOOST_NO_CXX11_EXPLICIT_CONVERSION_OPERATORS
@@ -186,7 +187,6 @@
 #define BOOST_NO_CXX11_CHAR16_T
 #define BOOST_NO_CXX11_CHAR32_T
 #define BOOST_NO_CXX11_CONSTEXPR
-#define BOOST_NO_CXX11_DECLTYPE_N3276
 #define BOOST_NO_CXX11_NOEXCEPT
 #define BOOST_NO_CXX11_UNICODE_LITERALS
 #define BOOST_NO_SFINAE_EXPR
--- boost_1_55_0.orig/libs/config/test/no_decltype_n3276_fail.cpp	2012-07-15 09:59:05.000000000 -0600
+++ boost_1_55_0/libs/config/test/no_decltype_n3276_fail.cpp	2014-02-22 23:27:19.034446382 -0700
@@ -32,6 +32,6 @@
 
 int main( int, char *[] )
 {
-   return boost_no_decltype_n3276::test();
+   return boost_no_cxx11_decltype_n3276::test();
 }
 
--- boost_1_55_0.orig/libs/config/test/no_decltype_n3276_pass.cpp	2012-07-15 09:59:05.000000000 -0600
+++ boost_1_55_0/libs/config/test/no_decltype_n3276_pass.cpp	2014-02-22 23:27:55.433988777 -0700
@@ -27,11 +27,11 @@
 #ifndef BOOST_NO_CXX11_DECLTYPE_N3276
 #include "boost_no_decltype_n3276.ipp"
 #else
-namespace boost_no_decltype_n3276 = empty_boost;
+namespace boost_no_cxx11_decltype_n3276 = empty_boost;
 #endif
 
 int main( int, char *[] )
 {
-   return boost_no_decltype_n3276::test();
+   return boost_no_cxx11_decltype_n3276::test();
 }
 
