--- FindOpenJpeg.cmake.orig	2014-03-02 20:56:48.830410643 -0700
+++ FindOpenJpeg.cmake	2014-03-02 20:57:14.014094041 -0700
@@ -84,7 +84,7 @@
 
 
 # Locate the header files
-PREFIX_FIND_INCLUDE_DIR (OpenJpeg openjpeg.h OpenJpeg_include_paths)
+find_path (OPENJPEG_INCLUDE_DIR openjpeg.h)
 
 # If the headers were found, add its parent to the list of lib directories
 if (OPENJPEG_INCLUDE_DIR)
@@ -97,15 +97,13 @@
 # Locate the OpenEXR library
 set (OpenJpeg_libvars "")
 set (OpenJpeg_cachevars "")
-PREFIX_FIND_LIB (OpenJpeg openjpeg
-  OpenJpeg_library_paths OpenJpeg_libvars OpenJpeg_cachevars)
+find_library (OPENJPEG_LIBRARIES NAMES openjpeg)
 
 # Use the standard function to handle OPENEXR_FOUND
 FIND_PACKAGE_HANDLE_STANDARD_ARGS (OpenJpeg DEFAULT_MSG
   OPENJPEG_INCLUDE_DIR ${OpenJpeg_libvars})
 
 if (OPENJPEG_FOUND)
-  set (OPENJPEG_LIBRARIES "")
   foreach (tmplib ${OpenJpeg_libvars})
     list (APPEND OPENJPEG_LIBRARIES ${${tmplib}})
   endforeach ()
