--- CMakeLists.txt.orig	2014-02-23 15:30:57.979563804 -0700
+++ CMakeLists.txt	2014-02-23 15:31:57.138820071 -0700
@@ -24,6 +24,10 @@
   ADD_DEFINITIONS ( -pthread )
 ENDIF ()
 
+IF (WIN32)
+  ADD_DEFINITIONS ( -DPLATFORM_WINDOWS )
+ENDIF ()
+
 INCLUDE_DIRECTORIES ( Iex IexMath Imath Half config IlmThread IexTest ImathTest HalfTest )
 
 MACRO(GET_TARGET_PROPERTY_WITH_DEFAULT _variable _target _property _default_value)
