--- nmake.opt.orig	2006-06-14 18:12:02.000000000 -0600
+++ nmake.opt	2014-03-11 22:23:46.000000000 -0600
@@ -54,19 +54,19 @@
 #
 # Uncomment and edit following lines to enable JPEG support.
 #
-#JPEG_SUPPORT	= 1
-#JPEGDIR 	= d:/projects/jpeg-6b
-#JPEG_INCLUDE	= -I$(JPEGDIR)
-#JPEG_LIB 	= $(JPEGDIR)/Release/jpeg.lib
+JPEG_SUPPORT   = 1
+JPEGDIR        = $(LUX_WINDOWS_DEPS_ROOT)\x86\$(BUILD_CONFIGURATION)
+JPEG_INCLUDE   = -I$(JPEGDIR)\include
+JPEG_LIB       = $(JPEGDIR)\lib\jpeg.lib
 
 #
 # Uncomment and edit following lines to enable ZIP support
 # (required for Deflate compression and Pixar log-format)
 #
-#ZIP_SUPPORT	= 1
-#ZLIBDIR 	= d:/projects/zlib-1.2.1
-#ZLIB_INCLUDE	= -I$(ZLIBDIR)
-#ZLIB_LIB 	= $(ZLIBDIR)/zlib.lib
+ZIP_SUPPORT    = 1
+ZLIBDIR        = $(LUX_WINDOWS_DEPS_ROOT)\x86\$(BUILD_CONFIGURATION)
+ZLIB_INCLUDE   = -I$(ZLIBDIR)\include
+ZLIB_LIB       = $(ZLIBDIR)\lib\zlib1.lib
 
 #
 # Uncomment and edit following lines to enable ISO JBIG support
@@ -80,7 +80,7 @@
 # Uncomment following line to enable Pixar log-format algorithm
 # (Zlib required).
 #
-#PIXARLOG_SUPPORT = 1
+PIXARLOG_SUPPORT = 1
 
 #
 # Comment out the following lines to disable strip chopping
@@ -115,7 +115,9 @@
 # NOTE: /EHsc option required if you want to build the C++ stream API
 #
 OPTFLAGS =	/Ox /MD /EHsc /W3 /D_CRT_SECURE_NO_DEPRECATE
-#OPTFLAGS = 	/Zi 
+!IFDEF DEBUG
+OPTFLAGS = 	/Zi /MDd /EHsc /W3 /D_CRT_SECURE_NO_DEPRECATE
+!ENDIF
 
 #
 # Uncomment following line to enable using Windows Common RunTime Library
@@ -129,6 +129,9 @@
 # for the option reference.
 #
 MAKE	=	nmake /nologo
+!IFDEF DEBUG
+MAKE	=	nmake /nologo DEBUG=1
+!ENDIF
 CC	=	cl /nologo
 CXX	=	cl /nologo
 AR	=	lib /nologo
