--- visualc.hpp.orig	2018-01-09 18:53:33.545322500 +0100
+++ visualc.hpp	2018-04-25 21:47:26.304162300 +0200
@@ -243,6 +243,10 @@
 #     define BOOST_COMPILER_VERSION 11.0
 #   elif _MSC_VER < 1900
 #     define BOOST_COMPILER_VERSION 12.0
+#   elif _MSC_VER < 1910
+#     define BOOST_COMPILER_VERSION 14.0
+#   elif _MSC_VER < 1920
+#     define BOOST_COMPILER_VERSION 14.1
 #   else
 #     define BOOST_COMPILER_VERSION _MSC_VER
 #   endif
@@ -257,6 +261,6 @@
 #  if defined(BOOST_ASSERT_CONFIG)
 #     error "Unknown compiler version - please run the configure tests and report the results"
 #  else
-#     pragma message("Unknown compiler version - please run the configure tests and report the results")
+//#     pragma message("Unknown compiler version - please run the configure tests and report the results")
 #  endif
 #endif
